sdcm/nemesis.py

```diff
@@ -2636,3 +2636,38 @@ def disrupt(self):
 #     @log_time_elapsed_and_status
 #     def disrupt(self):
 #         self.disable_disrupt_corrupt_then_scrub()
+
+
+class SlowHddNemesis(Nemesis):
+    def disrupt(self):
+        if isinstance(self.cluster, ScyllaAWSCluster):
```

Comment:
better to use `params.get('cluster_backend') == 'aws'`, could be that we would have other classes and still we are in AWS.

Reply:
Thanks, done :-)